tests/installation/partitioning_raid.pm

```diff
@@ -152,43 +149,60 @@ sub set_lvm {
     type_string "root";
     assert_screen 'volume-name-root';
     send_key $cmd{next};
-
-    # keep default
     send_key $cmd{next};
 
+    assert_screen 'volume-pick-fs-role';
     send_key "alt-o";    # Operating System
     send_key $cmd{next};
+    if (is_storage_ng) {
+        if (check_screen 'volume-empty-mount') {
+            # storage-ng does not suggest mount point for OS role volume,
+            # so mount the volume as root explicitely.
```

Comment:
s/explicitely/explicitly and please add soft-failure (https://bugzilla.suse.com/show_bug.cgi?id=1073854) as this is a bug. @okurz will also chase you for check_screen with non-zero timeout. Not to waste time, better to do assertion on both tags (volume-empty-mount and volume-mount-as-root), and if match_has_tag returns true for volume-empty-mount add soft-failure and set mount point.

Reply:
Yes I will :)
And also after you are getting used to it, it resembles much more how humans think. Normally people would not just check the screen and be blind for the alternatives showing up at the same time

Reply:
OK, sorry for the vintage code & non-human coding practices :). Fixed & re-verified: http://assam.suse.cz/tests/263#step/partitioning_raid/198.